Stop the applications from making a network connection every time they run

--- src/update_check.h
+++ src/update_check.h
@@ -96,18 +96,4 @@
 
 void check_version(const char* this_version)
 {
-	char curr_version[256];
-    memset(curr_version, 0, sizeof(curr_version));
-	if (get_current_version(curr_version))
-	{
-		if (strcmp(curr_version, this_version)==0)
-			fprintf(stderr, "You are using Cufflinks v%s, which is the most recent release.\n", PACKAGE_VERSION);
-		else
-			fprintf(stderr, "Warning: Your version of Cufflinks is not up-to-date. It is recommended that you upgrade to Cufflinks v%s to benefit from the most recent features and bug fixes (http://cufflinks.cbcb.umd.edu).\n", curr_version);
-		
-	}
-	else 
-	{
-		fprintf(stderr, "Warning: Could not connect to update server to verify current version. Please check at the Cufflinks website (http://cufflinks.cbcb.umd.edu).\n");
-	}
 }
